ports/raspberrypi/boards/fwk_keyboard/mpconfigboard.mk

```diff
@@ -0,0 +1,64 @@
+USB_VID = 0x32AC
+USB_PID = 0x0013
```

Comment:
TODO: I think I want to rename the board and use a different PID.

Reply:
Changed to draft.